gpii/node_modules/gsettingsBridge/nodegsettings/nodegsettings_tests.js

```diff
@@ -65,26 +64,26 @@ var testIntegerValues = function () {
 /* Test Reading Values */
 var testReadingDoubleValues = function () {
     var ret = gsettings.get_gsetting(keyScreenmag, "mag-factor");
-    util.puts("The mag-factor is: " + ret);
+    console.log("The mag-factor is: " + ret);
```

Comment:
Yes,

should I just add some assertions or do you want me to rewrite all these tests to use jqUnit and so on?